addon/services/key-manager.js

```diff
@@ -91,23 +102,15 @@ export default Service.extend({
   },
 
   handleEvent(event) {
-    const isServiceDestroyed = get(this, 'isDestroyed') || get(this, 'isDestroying');
-
-    if (isServiceDestroyed) {
-      return false;
+    if (get(this, 'isDestroyed') || get(this, 'isDestroying')) {
+      return;
```

Comment:
Nit, but I'd prefer to be explicit about what we are returning. This returns `undefined` but is not clear.